services/agent/src/simcore_service_agent/modules/volumes_cleanup/_docker.py

```diff
@@ -0,0 +1,33 @@
+from collections import deque
+from contextlib import asynccontextmanager
+from typing import AsyncIterator
+
+from aiodocker import Docker
+from aiodocker.utils import clean_filters
+from aiodocker.volumes import DockerVolume
+from servicelib.docker_constants import PREFIX_DYNAMIC_SIDECAR_VOLUMES
+
+
+@asynccontextmanager
+async def docker_client() -> AsyncIterator[Docker]:
```

Comment:
Yes, the idea is to have one unique source of truth. If we want to make changes in how we recover this docker instance, it will be easier to refactor.